### PR TITLE
Add PR template + update doc

### PR DESCRIPTION
**Pull request by araffin**
I added a PR template to avoid asking again and again the same things...
I also fixed a small bug in the documentation.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<!--- You can use the syntax `closes #100` if this solves the issue #100 -->
+- [ ] I have raised an issue to propose this change ([required](https://github.com/hill-a/stable-baselines/blob/master/CONTRIBUTING.md) for new features and bug fixes)
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Documentation (update in the documentation)
+- [ ] Other
```
Comment (Miffyli) [inline .github/PULL_REQUEST_TEMPLATE.md]: I do not think "Other" is needed as one can leave nothing selected.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I've read the [CONTRIBUTION](https://github.com/hill-a/stable-baselines/blob/master/CONTRIBUTING.md) guide (**required**)
+- [ ] I have updated the [changelog](https://github.com/hill-a/stable-baselines/blob/master/docs/misc/changelog.rst) accordingly (**required**).
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
+- [ ] I have updated the documentation accordingly.
```
Comment (Miffyli) [inline .github/PULL_REQUEST_TEMPLATE.md]: How does this one differ from "My change requires a change to the documentation"? Wouldn't this last option cover it?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<!--- This Template is ad edited version of the one from https://github.com/evilsocket/pwnagotchi/ -->
```
Comment (Miffyli) [inline .github/PULL_REQUEST_TEMPLATE.md]: "ad edited" or "an edited"?

**Comment by Miffyli** [comment]
Ah yes, very handy feature! Also helps if ever there is a derp person going around and forgetting people to do certain things ^^'

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (araffin) [reply .github/PULL_REQUEST_TEMPLATE.md]: This is a checklist, so at first you say (when it is WIP) that is requires changes, and then when it is done, you can check this one.

**Pull request merged**